CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add `CONJUR_AUTHN_JWT_SERVICE_ID` & `JWT_TOKEN_PATH` environment variables as configuration to support authn-jwt 
+
```

Comment:
Can you add a link to this PR? [cyberark/conjur-api-go#124](https://github.com/cyberark/conjur-api-go/pull/124)

Reply:
Sure.  Link added